calcs/moon.js

```diff
@@ -65,21 +65,66 @@ module.exports = function (app, plugin) {
       app.debug('moon times:' + JSON.stringify(times, null, 2))
 
       return [
-        { path: 'environment.moon.fraction', value: illumination.fraction },
-        { path: 'environment.moon.phase', value: illumination.phase },
+        {
+          path: 'environment.moon.fraction',
+          value: Number(illumination.fraction)
+        },
+        { path: 'environment.moon.phase', value: Number(illumination.phase) },
         { path: 'environment.moon.phaseName', value: phaseName },
-        { path: 'environment.moon.angle', value: illumination.angle },
+        { path: 'environment.moon.angle', value: Number(illumination.angle) },
         { path: 'environment.moon.times.rise', value: times.rise || null },
         { path: 'environment.moon.times.set', value: times.set || null },
         {
           path: 'environment.moon.times.alwaysUp',
-          value: times.alwaysUp ? 'true' : 'false'
+          value: Boolean(times.alwaysUp ? 'true' : 'false')
         },
         {
           path: 'environment.moon.times.alwaysDown',
-          value: times.alwaysDown ? 'true' : 'false'
+          value: Boolean(times.alwaysDown ? 'true' : 'false')
```

Comment:
alwaysUp and alwaysDown are either `true` Boolean or `undefined` coming from suncalc. Here you convert them first to string and then to Boolean. `value: !!times.alwaysDown` would do that trick in a much more concise way.

https://stackoverflow.com/questions/29312123/how-does-the-double-exclamation-work-in-javascript